lib/internal/fs/utils.js

```diff
@@ -155,6 +155,10 @@ function Stats(
 }
 
 Stats.prototype._checkModeProperty = function(property) {
+  if (isWindows && (property === S_IFIFO || property === S_IFBLK ||
+    property === S_IFSOCK)) {
```

Comment:
These three are in fact all undefined on Windows, but I figured it looks more readable if I list them like that.